Makefile

```diff
@@ -22,7 +23,7 @@ migrate:
 
 dao:
 		@command -v gentool >/dev/null 2>&1 || (echo "Installing gentool..." && go install gorm.io/gen/tools/gentool@latest)
-		gentool -dsn ${DB_DSN} -fieldNullable -fieldWithIndexTag -fieldWithTypeTag -fieldSignable -onlyModel -outPath "./database/dao" -modelPkgName "dao"
+		gentool -dsn ${BB_DSN_GEN} -fieldNullable -fieldWithIndexTag -fieldWithTypeTag -fieldSignable -onlyModel -outPath "./database/dao" -modelPkgName "dao"
```

Comment:
@bhattaraibishal50 can you make it like of migrate command, where DSN is used based on env provided

Reply:
you mean to add the localhost and port 33066 to .env

Reply:
@sawal-timsina 
Or mean to execute via docker instead of installing into local machine ?